api/src/lib.rs

```diff
@@ -2273,6 +2243,198 @@ ffi_export!(
     }
 );
 
+ffi_export!(
+    fn Java_com_horizen_schnorrnative_SchnorrKeysSignaturesList_nativeKeysRootHash(
```

Comment:
We changed the name, Java-side, from SchnorrKeysSignaturesList to ValidatorKeysUpdatesList. That class is not tested otherwise we would've spotted the bug.
Please let's change the name Java side and let's add some Java tests for this class too